dll/win32/shell32/shelldesktop/CChangeNotify.h

```diff
@@ -132,14 +132,14 @@ class CChangeNotify : public CWorker
     BOOL ShouldNotify(LPDELITICKET pTicket, LPNOTIFSHARE pShared);
     BOOL DoNotify(LPHANDBAG pHandBag, LPDELITICKET pTicket, LPNOTIFSHARE pShared);
 
-    LRESULT OnBang(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL& bHandled);
+    LRESULT OnReg(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL& bHandled);
     LRESULT OnUnReg(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL& bHandled);
     LRESULT OnDelivery(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL& bHandled);
     LRESULT OnSuspendResume(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL& bHandled);
     LRESULT OnRemoveByPID(UINT uMsg, WPARAM wParam, LPARAM lParam, BOOL& bHandled);
 
     BEGIN_MSG_MAP(CChangeNotify)
-        MESSAGE_HANDLER(WM_NOTIF_BANG, OnBang)
+        MESSAGE_HANDLER(WM_NOTIF_BANG, OnReg)
```

Comment:
Then, should this message be called `WM_NOTIF_REG` as well?

Reply:
Renamed.